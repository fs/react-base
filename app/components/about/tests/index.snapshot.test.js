import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import About from '../';

describe('About', () => {
  let props;
  const renderComponent = () => renderer.create(
    <MemoryRouter>
      <About { ...props } />
    </MemoryRouter>
  );

  beforeEach(() => {
    props = {
      match: {
        url: '/about'
      }
    };
  });

  it('renders correctly', () => {
    expect(renderComponent().toJSON()).toMatchSnapshot();
  });
});
