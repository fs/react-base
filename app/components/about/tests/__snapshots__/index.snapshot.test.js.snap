// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders correctly 1`] = `
<div
  className="container"
>
  <article>
    <h1>
      About
    </h1>
    <p>
      Kick-start your new web application based on React and Redux technologies.
    </p>
    <a
      href="/about/extended/test_id"
      onClick={[Function]}
    >
      show details...
    </a>
  </article>
</div>
`;
